src/_round.ts

```diff
@@ -0,0 +1,21 @@
+export function _round(roundingFn: (value: number) => number) {
```

Comment:
like in Lodash, the right name here is probably something like createRound or as a higher-order function: `withPrecision`
```suggestion
export function _withPrecision(roundingFn: (value: number) => number) {
```